about.html

```diff
@@ -0,0 +1,349 @@
+              FIND THE BEST COMIC WHEREVER YOU ARE
+            </h2>
+          </section>
+          <section class="about-header-text-description-container">
+            <p class="about-header-text-description font">
+              Discover the best comics, matches to your comdey taste.
+            </p>
+          </section>
+          <section class="contact-paragraph-div">
+            <h5 class="contact-paragraph font">
+              Please contact us per Email for any further questions about
+              ComedyBurst
+            </h5>
+          </section>
+          <section class="email-link-div">
+            <a class="email-link" href="#">ComedyBurst@comic.org</a>
+          </section>
+        </div>
+        <div class="about-page-header-body-desk for-desktop">
+          <section class="about-header-text-div-desk">
+            <h1 class="about-header-text-body-desk font">
+              FIND THE BEST COMIC WHEREVER YOU ARE
+            </h1>
+          </section>
+          <section class="about-header-text-description-container-desk">
+            <p class="about-header-text-description-desk font">
+              ComdeyBurst has been a trusted home for specials since 2017. From
+              day one, we’ve set about making it as easy, fun and fair as
+              possible for you to see your favorite artists live. Right now,
+              more than 60 million comic fans across the globe use ComedyBurst to
+              track their favorite artists, discover events and buy tickets
+              with joy.
+              <br />
+              We’ve always put fans first. That means no spam about things you
+              don’t care about, and no inflated ticket prices. Live event should
+              work for everyone, and we work hard to empower our community and
+              cut out the BS along the way. Nothing but good vibes from here.
+              <br />
+              Best of all, ComedyBurst is totally free. Sign up and track comics via
+              our website, or download our iPhone and Android apps for event
+              alerts on the go. Either way, you’ll be the first to know about
+              the best shows in your area. We’ll see you down the front.
+            </p>
+          </section>
+          <section class="desk-contact-paragraph-div">
+            <h5 class="contact-paragraph font">
+              Please contact us per Email for any further questions about
+              ComedyBurst
+            </h5>
+          </section>
+          <section class="email-link-div-desk">
+            <span class="font"
+              ><a class="email-link-desk" href="#">ComedyBurst@comic.org</a>
+              ></span
+            >
+          </section>
+        </div>
+      </section>
+      <section id="main">
+        <div class="about-main-header">
+          <h1 class="font">See Past Events</h1>
+          <section class="about-header-red-line font">____</section>
```

Comment:
- [ ] Kindly note that the `h1` tag should be used only once on each page, kindly use another `hx`.

Reply:
Done 👍 @Ofarouq310